js/messages.php

```diff
@@ -541,6 +541,7 @@ function () {
 $js_messages['strYes'] = __('Yes');
 $js_messages['strCopyEncryptionKey'] = __('Do you want to copy encryption key?');
 $js_messages['strEncryptionKey'] = __('Encryption key');
+$js_messages['HexConversionInfo'] = __('HEX function will treat the integer as String while calculating hexadecimal value');
```

Comment:
`String` should not have a capital letter ?
```suggestion
/* l10n: Please explain here to translators what they are translating 
$js_messages['HexConversionInfo'] = __('HEX function will treat the integer as string while calculating hexadecimal value');
```